lib/core/core.nit

```diff
@@ -33,3 +39,7 @@ import error
 import re
 import bytes
 import fixed_ints
+
+redef class Object
+	private init nitunit do end
```

Comment:
There is a special magic undocumented constructor, called `intern`, that does just that

~~~
var x = new MyClass.intern
~~~